bin/db-backup.sh

```
@@ -0,0 +1,41 @@
+if [ "$BACKUP_OVERWRITE" = "true" ]; then
+  FILENAME="backup_latest.sql.gz"
+else
+  FILENAME="backup_${TIMESTAMP}.sql.gz"
+fi
+
+TEMP_FILE="/tmp/${FILENAME}"
+
+trap 'rm -f "${TEMP_FILE}"' EXIT
+
+echo "Starting backup for ${POSTGRES_DB} to ${TEMP_FILE}..."
+set -o pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)